Strata/DDM/Format.lean

```diff
@@ -588,27 +592,27 @@ instance FunctionDecl.instToStrataFormat : ToStrataFormat FunctionDecl where
 
 namespace MetadataArgType
 
-protected def mformat : MetadataArgType → StrataFormat
-  | .num => mf!"Num"
-  | .ident => mf!"Ident"
-  | .bool => mf!"Bool"
-  | .opt tp => mf!"Option {tp.mformat |>.ensurePrec (appPrec + 1)}" |>.setPrec appPrec
+private protected def mformat : MetadataArgType → StrataFormat
+| .num => mf!"Num"
+| .ident => mf!"Ident"
+| .bool => mf!"Bool"
+| .opt tp => mf!"Option {tp.mformat |>.ensurePrec (appPrec + 1)}" |>.setPrec appPrec
```

Comment:
Oh nice, reducing indentation makes total sense here. Is this done automatically somehow everywhere or did you just do it yourself here? We could consider putting up a style guide.